api/hypershift/v1beta1/hostedcluster_conditions.go

```diff
@@ -132,9 +132,9 @@ const (
 	// A failure here may require external user intervention to resolve.
 	ValidAWSIdentityProvider ConditionType = "ValidAWSIdentityProvider"
 
-	// ValidAWSKMSConfig indicates whether the AWS KMS role and encryption key are valid and operational
-	// A failure here indicates that the role or the key are invalid, or the role doesn't have access to use the key.
-	ValidAWSKMSConfig ConditionType = "ValidAWSKMSConfig"
```

Comment:
this would be a breaking change, we need to add a new condition for azure. Or a generic one but keep this one in deprecated state.

Reply:
https://gitlab.cee.redhat.com/search?search=ValidAWSKMSConfig&nav_source=navbar&project_id=12342&group_id=5301&search_code=true&repository_ref=master